### DISCUSSION TITLE
Welcome to solidity-cookbook Discussions!

### DISCUSSION THREAD

**Discussion by pnpancholi** [PradhumnaPancholi/solidity-cookbook#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lvnacy** [comment]
Hello, I’m @nonsensecodes, better known as Nonsense in the BanklessDAO. I’m predominantly a JS developer, though I’ve learned some Go and Rust and am currently working to learn Solidity. I just started a Solidity Learning Workgroup for DAO members, and hope to contribute to this cookbook through understanding what works and what doesn’t as the workgroup progresses.
